distros/kubernetes/nvsentinel/charts/fault-remediation/files/log-collector-job.yaml

```diff
@@ -61,6 +61,12 @@ spec:
               value: {{ .Values.logCollector.enableGcpSosCollection | quote }}
             - name: ENABLE_AWS_SOS_COLLECTION
               value: {{ .Values.logCollector.enableAwsSosCollection | quote }}
+            - name: MOCK_MODE
+              value: {{ .Values.logCollector.mockMode | default "false" | quote }}
+            - name: MOCK_EXIT_CODE
+              value: {{ .Values.logCollector.mockExitCode | default "0" | quote }}
+            - name: MOCK_SLEEP_DURATION
+              value: {{ .Values.logCollector.mockSleepDuration | default "5" | quote }}
```

Comment:
We don't have to add default here and in the values, let's keep it in the values file